.azure/pipelines/quarantined-pr.yml

```diff
@@ -1,9 +1,46 @@
-# We want to run quarantined tests on master as well as on PRs
+#
+# See https://docs.microsoft.com/en-us/vsts/pipelines/yaml-schema for details on this file.
+#
+
+# Configure which branches trigger builds
+# We want to run quarantined tests on release/5.0 and master as well as on PRs
 trigger:
   batch: true
   branches:
     include:
     - master
+    - release/5.0
+  paths:
+    exclude:
+    - .github/*
+    - .vscode/*
+    # Exclude only a few specific Markdown files because some of them ship.
+    - docs/*
+    - CODE-OF-CONDUCT.md
+    - CONTRIBUTING.md
+    - LICENSE.TXT
+    - README.md
+    - SECURITY.md
+    # Do not ignore the THIRD-PARTY-NOTICES.TXT file because it ships.
+
+# Run PR validation on branches that include Helix tests
+pr:
+  autoCancel: true
+  branches:
+    include:
+    - release/5.0
+    - master
+  paths:
+    exclude:
+    - .github/*
+    - .vscode/*
+    - '**/*.md'
+    - CODE-OF-CONDUCT.md
+    - CONTRIBUTING.md
+    - LICENSE.TXT
+    - README.md
+    - SECURITY.md
+    - THIRD-PARTY-NOTICES.TXT
 
 schedules:
```

Comment:
Should I remove the `schedules`❔ Rolling and PR should be more than frequent enough.

Reply:
The analytics only count master builds right?  These checks run on PR but they don't count PR failures for analytics since then pretty much  every test would be considered flaky if they ever fail as part of working on features in PRs...  So I think we still need the scheduled runs to get enough volume to be able to unquarantine tests.

Reply:
> The analytics only count master builds right? 

AzDO shows every build of the pipeline in the analytics view by default. But, it can be filtered down to whatever branch you choose.

More generally, these six extra builds will almost always (not always due to batching) just duplicate rolling builds and those already occur after every checkin. We don't do anything like this with the aspnetcore-ci pipeline and I'm not sure the extra 6 builds per day are worth the resources consumed.

Reply:
Given @Pilchie's [response above](https://github.com/dotnet/aspnetcore/pull/27484#discussion_r517692304), I'm going to leave these extra 6 builds/day alone.